libs/design/src/molecules/paginator/paginator.component.ts

```diff
@@ -0,0 +1,100 @@
+import { Component, Input, Output, EventEmitter, OnInit, HostBinding, ElementRef, HostListener } from '@angular/core';
+
+import { daffColorMixin, DaffPalette, DaffColorable } from '../../core/colorable/colorable';
+
+/**
+ * An _elementRef is needed for the Colorable mixin
+ */
+class DaffPaginatorBase {
+  constructor(public _elementRef: ElementRef) {}
+}
+
+const _daffPaginatorBase = daffColorMixin(DaffPaginatorBase)
+
+const visiblePageRange = 2;
+
+@Component({
+  selector: 'daff-paginator',
+  styleUrls: ['./paginator.component.scss'],
+  templateUrl: './paginator.component.html'
+})
+export class DaffPaginatorComponent extends _daffPaginatorBase implements OnInit, DaffColorable {
+  @HostBinding('class.daff-paginator') class = true;
+  @HostBinding('attr.role') role = 'navigation';
+  @Input() color: DaffPalette;
+  @Input() id: string;
```

Comment:
I think instead we should grab the value of `aria-label` off the host-binding if it exists.

Reply:
done